webapp/src/main/java/org/apache/atlas/web/rest/AuthREST.java

```diff
@@ -164,6 +172,47 @@ public ServicePolicies downloadPolicies(@PathParam("serviceName") final String s
         }
     }
 
+    @POST
+    @Path("refreshcache")
+    @Timed
+    public void refreshCache(@QueryParam("policies") Boolean policies,
+                             @QueryParam("roles") Boolean roles,
+                             @QueryParam("groups") Boolean groups) {
+        AtlasPerfTracer perf = null;
+
+        try {
+            if (AtlasPerfTracer.isPerfTraceEnabled(PERF_LOG)) {
+                perf = AtlasPerfTracer.getPerfTracer(PERF_LOG, "AuthREST.refreshCache(policies="+policies+", roles="+roles+", groups="+groups+")");
+            }
+
+            if (policies == null && roles == null && groups == null) {
+                AtlasAuthorizationUtils.refreshCache(true, true, true);
+
+            } else {
+                boolean refreshPolicies = false;
```

Comment:
nit: by default, if not set the values are always false for policies, roles and groups if we take primitives, rather than objects ( Boolean )

Reply:
to simplify API usage, in case no queryParam is passed, we assume refresh all
If we use primitives we will get false even if client doesn't pass any query param

So used Object to know whether it is null or explicit false